disperser/dataapi/metrics_handlers.go

```diff
@@ -17,19 +17,21 @@ const (
 )
 
 func (s *server) getMetric(ctx context.Context, startTime int64, endTime int64, limit int) (*Metric, error) {
-	operators, err := s.subgraphClient.QueryOperatorsWithLimit(ctx, limit)
-	if err != nil {
-		return nil, err
-	}
-
 	blockNumber, err := s.transactor.GetCurrentBlockNumber(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get current block number: %w", err)
 	}
-	totalStake, err := s.calculateTotalStake(operators, blockNumber)
+	operatorState, err := s.chainState.GetOperatorState(ctx, uint(blockNumber), []core.QuorumID{core.QuorumID(0)})
```

Comment:
@gpsanant we are counting on this to get the stakes onchain, let me know if it's not reliable, we'd otherwise merge this PR